src/ipc/capnp/mining.capnp

```diff
@@ -17,10 +17,11 @@ interface Mining $Proxy.wrap("interfaces::Mining") {
     isInitialBlockDownload @1 (context :Proxy.Context) -> (result: Bool);
     getTip @2 (context :Proxy.Context) -> (result: Common.BlockRef, hasResult: Bool);
     waitTipChanged @3 (context :Proxy.Context, currentTip: Data, timeout: Float64) -> (result: Common.BlockRef);
-    createNewBlock @4 (scriptPubKey: Data, options: BlockCreateOptions) -> (result: BlockTemplate);
-    processNewBlock @5 (context :Proxy.Context, block: Data) -> (newBlock: Bool, result: Bool);
-    getTransactionsUpdated @6 (context :Proxy.Context) -> (result: UInt32);
-    testBlockValidity @7 (context :Proxy.Context, block: Data, checkMerkleRoot: Bool) -> (state: BlockValidationState, result: Bool);
+    waitFeesChanged @4 (context :Proxy.Context, currentTip: Data, feeThreshold: Int64, options: BlockCreateOptions, timeout: Float64) -> (result: Bool);
```

Comment:
a87589abfde4f2f1dfba0f3f1f5745f7acfef365: @ryanofsky or should I make this `@9` and leave the other numbers untouched?

Reply:
> [a87589a](https://github.com/bitcoin/bitcoin/commit/a87589abfde4f2f1dfba0f3f1f5745f7acfef365): @ryanofsky or should I make this `@9` and leave the other numbers untouched?

You can but it shouldn't matter, as long as nothing outside the build is depending on the interface. So far I've just renumbered everything like your current PR does. I even have a hacky renumbering script `capnp-renum`, that I pipe iinto from vim to renumber structs and interfaces:

```python
#!/usr/bin/env python

import sys
import re

next = 0 if len(sys.argv) < 2 else int(sys.argv[1])

def f(m):
  global next
  next += 1
  return "@" + str(next - 1)

sys.stdout.write(re.sub(r"@\d+", f, sys.stdin.read()))

```

Reply:
Ok, well the numbering here doesn't conflict with #30955 and there's no other proposed interface change pending. So I'll keep the renumbering here.